Sources/Aardvark/BugReporting/LogStoreAttachmentGenerator.swift

```diff
@@ -43,12 +43,14 @@ public final class LogStoreAttachmentGenerator: NSObject {
     /// - parameter messageFormatter: The formatter used to format messages in the logs attachment.
     /// - parameter includeLatestScreenshot: Whether an attachment should be generated for the last screenshot in the
     /// log store, if one exists.
+    /// - parameter numberOfErrorsInHighlights: The number of errors to include in the highlights for the log store.
     /// - parameter completionQueue: The queue on which the completion should be called.
     /// - parameter completion: The completion to be called once the attachments have been generated.
     public static func attachments(
         for logStore: ARKLogStore,
         messageFormatter: ARKLogFormatter = ARKDefaultLogFormatter(),
         includeLatestScreenshot: Bool,
+        numberOfErrorsInHighlights: Int = 3,
```

Comment:
I think for now we'll leave it as-is. This should really be set based on the app's baseline non-fatal rate. For apps that are very strict about non-fatals, this could be 1 or 2. For apps that are a bit noisy, they might go more like 10.